docs/guidelines-for-reviewing-datascience-code.md

```diff
@@ -1,25 +1,23 @@
-# Guidelines for Reviewing Data Science Code
+# 10 Principles for Reviewing Data Science Code
```

Comment:
Comment: I suggested some of these are too general and should be removed, so here are some Data Science specific review principles to consider instead:

* Use Code Quality tools like flake8-nb, black-nb, etc.
* Make sure credentials and  external data access is managed correctly
* Are the train/test data sets separated correctly for your ML task?
* Is the data management handled in a universal way? (no hard coding directories)    
* Is the appropriate evaluation metric being used?

Reply:
The suggestions are very helpful. I will edit this document incorporating all of these 👍

Reply:
You can probably also expand on:

* Are the dependencies in code or environments used to run the code appropriately tracked ? eg: via Pipfile